app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -476,6 +479,23 @@ class BrowserTabViewModelTest {
         assertEquals(false, loadingViewState().isLoading)
     }
 
+    @Test
+    fun whenProgressChangesToFinishedAndImprovedTabUxVariantActiveThenUpdateWebViewPreview() {
+        whenever(mockVariantManager.getVariant()).thenReturn(Variant("mx", 1.0, listOf(VariantManager.VariantFeature.TabSwitcherGrid)))
+        updateUrl("https://example.com", "https://example.com", true)
+        testee.progressChanged(100)
+        val command = captureCommands().lastValue as Command.GenerateWebViewPreviewImage
+        assertFalse(command.forceImmediate)
+    }
+
+    @Test
+    fun whenProgressChangesToFinishedAndImprovedTabUxVariantNotActiveThenDoNotUpdateWebViewPreview() {
```

Comment:
Optional: As above. Can we change it to `whenProgressChangesToFinishedAndImprovedTabUxVariantNotActiveThenWebViewPreviewNotGenerated`?